categories.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
```

Comment:
The assignment mentions that the component library css should be used when building the macro and mini projects. Are you planning to add it to your quiz project?

Reply:
Updated the code to make use of component from library